<div align="right"><font size=+1 color="red">สื่อดิจิทัลเพื่อการศึกษาอาชีวศึกษา</font><br>หลักสูตรประกาศนียบัตรวิชาชีพชั้นสูง ประเภทวิชาอุตสาหกรรม สาขาวิชาเทคนิคเครื่องกล </div>

# <center><font color="blue"> 30000-1304 วิทยาศาสตร์งานเครื่องกลและการผลิต<br>Science for Mechanical and Production Works (2-2-3)</center>

<div align="right">บำรุง พ่วงเกิด PhD (Swansea, UK)<br>วิศวกรรมเครื่องกล สจล</div>

## เนื้อหารายวิชา

<font size=+1>
    
* กลศาสตร์วิศวกรรม (หน่วยที่ 1-11) ผู้สอน: สยาม สงวนรัมย์
* คุณสมบัติของสสาร (หน่วยที่ 12-15) ผู้สอน: บำรุง พ่วงเกิด
* ความร้อนและการถ่ายเทความร้อน (หน่วยที่ 16-17) ผู้สอน: บำรุง พ่วงเกิด
* ปิโตรเลียมและผลิตภัณฑ์ (หน่วยที่ 18) ผู้สอน: บำรุง พ่วงเกิด

# หน่วยที่ 12 - สมบัติของสสาร ภาค I <br> (Properties of Matters Part I)

## สสาร (Matters)

> **เอกสารอ้างอิง** John Bird 
[_Science and Mathematics for Engineering_ 6th Edition](https://www.amazon.com/Science-Mathematics-Engineering-John-Bird-ebook/dp/B07YNYFVJT/), [Routledge, 2020](https://routledgetextbooks.com/textbooks/9780367204747/).


ก่อนหน้านี้มีความคิดว่าสสาร (Matter) คือสิ่งที่ต้องการที่อยู่และมีน้ำหนัก และใช้เวลาสักครู่ที่นักวิทยาศาสตร์จะรู้ว่าอากาศและแก๊สก็เป็นสสารด้วย

นิยามทางวิทยาศาสตร์ของสสารคือสิ่งที่ต้องการที่อยู่ (Space) และมี__มวล (mass)__

มีการพบว่าสสารมีความไม่ต่อเนื่อง (not continuous) ทำให้เกิดทฤษฎีโมเลกุลของสสาร (Molecular theory of matters) และทฤษฎีอะตอมของสสาร (Atomic theory of matter) เกิดขึ้น โดยบอกว่าสสารประกอบไปด้วยอนุภาคขนาดเล็กเรียกว่าโมเลกุล (Molecules) และอะตอม (Atoms) โดยเคยมีความคิดว่าสสารประกอบด้วยอะตอมเป็นหน่วยที่เล็กที่สุดและไม่สามารถแบ่งแยกอะตอมได้ แต่จากการทดลองได้แสดงว่าอะตอมประกอบด้วยอนุภาคที่เล็กกว่าอาทิ โปรตอน (Protons) นิวตรอน (Neutrons) และอิเล็กตรอน (Electrons) 

พืจารณาถึงความแบ่งแยกต่อไปไม่ได้ พบว่าโปรตรอน และนิวตรอนประกอบด้วยอนุภาคที่แบ่งแยกต่อไปไปไม่ได้เรียกว่าควาก (Quarks) นอกจากนี้ยังมีการค้นพบอนุภาคที่ไม่เป็นส่วนหนึ่งของอะตอมอีกด้วย

<img src="https://i.pinimg.com/564x/59/bf/dc/59bfdc6d93eba5da364f7bb95ef144e2.jpg">

### ธาตุ อะตอม โมเลกุล และสารประกอบ

เรามีสาร (Substances) อยู่จำนวนมาก แต่ละสารจะประกอบไปด้วยวัสดุ (Materials) แบบพื้นฐานชนิดหนึ่งหรือหลายชนิด ที่เรียกว่า[ธาตุ (Elements)](https://th.wikipedia.org/wiki/ธาตุ)

> ธาตุ (Elements) คือสารที่ไม่สามารถแยกออกจากกันด้วยขบวนการทางเคมี

มีธาตุในธรรมชาติอยู่ 92 ธาตุ และเกิดจากการสังเคราะห์ 13 ธาตุ

ตัวอย่างของธาตุและสัญลักษณ์ของธาตุ

[ไฮโดรเจน H](https://th.wikipedia.org/wiki/ไฮโดรเจน)
[ฮีเลียม He](https://th.wikipedia.org/wiki/ฮีเลียม)
[คาร์บอน C](https://th.wikipedia.org/wiki/คาร์บอน)
[ไนโตรเจน N](https://th.wikipedia.org/wiki/ไนโตรเจน)
[ออกซิเจน O](https://th.wikipedia.org/wiki/ออกซิเจน)
[โซเดียม Na](https://th.wikipedia.org/wiki/โซเดียม)
[แมกนีเซียม Mg](https://th.wikipedia.org/wiki/แมกนีเซียม)
[อะลูมิเนียม Al](https://th.wikipedia.org/wiki/อะลูมิเนียม)
[ซิลิคอน Si](https://th.wikipedia.org/wiki/ซิลิคอน)
[ฟอสฟอรัส P](https://th.wikipedia.org/wiki/ฟอสฟอรัส)
[กำมะถัน S](https://th.wikipedia.org/wiki/กำมะถัน)
[โปรแทสเซียม K](https://th.wikipedia.org/wiki/โพแทสเซียม)
[แคลเซียม Ca](https://th.wikipedia.org/wiki/แคลเซียม)
[เหล็ก Fe](https://th.wikipedia.org/wiki/เหล็ก)
[นิกเกิล Ni](https://th.wikipedia.org/wiki/นิกเกิล)
[ทองแดง Cu](https://th.wikipedia.org/wiki/ทองแดง)
[สังกะสี Zn](https://th.wikipedia.org/wiki/สังกะสี)
[เงิน Ag](https://th.wikipedia.org/wiki/เงิน_(โลหะ))
[ดีบุก Tn](https://th.wikipedia.org/wiki/ดีบุก)
[ทองคำ Au](https://th.wikipedia.org/wiki/ทองคำ)
[ปรอท Hg](https://th.wikipedia.org/wiki/ปรอท)
[ตะกั่ว Pb](https://th.wikipedia.org/wiki/ตะกั่ว)
[ยูเรเนียม U](https://th.wikipedia.org/wiki/ยูเรเนียม)

#### อะตอม (Atoms)

![อะตอม](https://upload.wikimedia.org/wikipedia/commons/thumb/2/23/Helium_atom_QM.svg/800px-Helium_atom_QM.svg.png)

ธาตุประกอบด้วยองค์ประกอบย่อยเรียกว่า__อะตอม__

> อะตอมคือส่วนที่เล็กที่สุดของธาตุซึ่งมีส่วนในการเปลี่ยนแปลงทางเคมีและยังคงคุณสมบัติความเป็นธาตุอยู่

แต่ละธาตุประกอบด้วยอะตอมที่แตกต่างกันออกไป

__ทฤษฎีอะตอม__

แบบจำลองของอะตอมอาจมองว่ามีลักษณะคล้ายวงโคจรของระบบสุริยจักรวาล มี
[นิวเคลียส (Nucleus)](https://th.wikipedia.org/wiki/นิวเคลียสของอะตอม) อยู่ตรงกลาง มีอนุภาคที่มีประจุลบที่ชื่อว่า[อิเล็กตรอน (Electrons)](https://th.wikipedia.org/wiki/อิเล็กตรอน)
โคจรรอบอยู่ เป็นวงโคจรที่เป็นแถบคงที่เรียกว่าเป็นชั้น (Shells)

นิวเคลียสหรือใจกลางประกอบด้วยอนุภาคประจุบวกที่เรียกว่า[โปรตอน (Protons)](https://th.wikipedia.org/wiki/โปรตอน) และอนุภาคที่ไม่มีประจุอยู่ด้วยกันเรียกว่า[นิวตรอน (Neutrons)](https://th.wikipedia.org/wiki/นิวตรอน)

อิเล็กตรอนมีมวลน้อยมากเมื่อเทียบกับโปรตอนและนิวตรอน อะตอมหนึ่งมีความเป็นกลางทางไฟฟ้า นั่นคือประกอบด้วยอิเล็กตรอนและโปรตอนจำนวนเท่ากัน

จำนวนโปรตอนในอะตอมมีชื่อเรียกว่า[เลขอะตอม (Atomic Number)](https://th.wikipedia.org/wiki/เลขอะตอม) ของธาตุที่มีอะตอมเป็นส่วนประกอบ การจัดเรียงของธาตุตามลำดับของเลขอะตอมรู้จักกันในนามของ[ตารางธาตุ (Periodic Table)](https://th.wikipedia.org/wiki/ตารางธาตุ)

![ตารางธาตุ](https://upload.wikimedia.org/wikipedia/commons/thumb/8/84/Periodic_table.svg/1920px-Periodic_table.svg.png)

อะตอมที่ซับซ้อนน้อยที่สุดคืออะตอมของไฮโดรเจน โดยมี 1 อิเล็กตรอนโคจรรอบนิวเคลียส และมี 1 โปรตอนในนิวเคลียส เลขอะตอมของไฮโดรเจนจึงคือ 1 ในขณะที่ฮีเลียมมีเลขอะตอมเป็น 2 มี 2 อเล็กตรอนโคจรรอบนิวเคลียสโดยที่อิเล็กตรอนของทั้งสองอยู่บนชั้นเดียวกัน และมีระยะห่างจากนิวเคลียสเท่ากัน

![อะตอม](U12pics/atoms.jpg)

ชั้นแรกของวงโคจรของอิเล็กครอนสามาถบรรจุได้ 2 อิเล็กตรอนเท่านั้น ชั้นที่ 2 บรรจุได้ 8 อิเล็กตรอนเท่านั้น ชั้นที่ 3 บรรจุได้ 8 อิเล็กตรอนเท่านั้น ดังนั้นอะลูมิเนียมอะตอมมี 13 อิเล็กตรอนโคจรรอบนิวเคลียสจึงจัดเรียงตัวตามแผนภาพด้านบน

#### โมเลกุล (Molecules)

เมื่อหลายธาตุมารวมกัน อะตอมจะเชื่อมกันทำให้เกิดหน่วยใหม่ของสาร (Substance) เป็นกลุ่่มอิสระที่รวมตัวมีพันธะต่อกันเรียกว่า[โมเลกุล (Molecules)](https://th.wikipedia.org/wiki/โมเลกุล)

> โมเลกุลคือส่วนที่เล็กที่สุดของสาร ที่แยกมาแล้วยังคงสภาพเดิม

ทุกโมเลกุลมีสารเหมือนกันหมด ดังนั้นอะตอมและโมเลกุลคือหน่วยพื้นฐานของสสารนั่นเอง

#### สารประกอบ (Compounds)

เมื่อหลายธาตุมารวมกันทางเคมีอะตอมเชื่อมต่อกันเป็นโมเลกุลของสารใหม่เรียกว่า[สารประกอบ (Compounds)](ttps://th.wikipedia.org/wiki/สารประกอบ) 

> สารประกอบคือสารใหม่ประกอบด้วยสองธาตุหรือมากกว่ารวมกันทางเคมีทำให้มีคุณสมบัติถูกเปลี่ยนไป

ตัวอย่างธาตุไฮโดรเจนและออกซิเจนรวมกันทางเคมีเป็นสารประกอบน้ำ ซึ่งน้ำมีคุณสมบัติต่างไปจากไฮโดรเจนและออกซิเจน

ตัวอย่างของสารประกอบ

* [น้ำ](https://th.wikipedia.org/wiki/น้ำ) $\rm{H_2 O}$
* [เกลือแกงหรือโซเดียมคลอไรด์](https://th.wikipedia.org/wiki/โซเดียมคลอไรด์) $\rm{NaCl}$
* [คาร์บอนไดออกไซด์](https://th.wikipedia.org/wiki/คาร์บอนไดออกไซด์) $\rm{CO_2}$
* [จุนสีหรือคอปเปอร์ซัลเฟต](https://th.wikipedia.org/wiki/จุนสี) $\rm{CuSO_4}$

### สารผสม สารละลาย สารแขวนลอย และการทำละลาย

#### สารเนื้อผสม (Mixtures)

> สารเนื้อผสม เป็นการรวมของสารที่ไม่ได้รวมกันทางเคมี

สารเนื้อผสมยังคงคุณสมบัติของสารองค์ประกอบอยู่ ทั้งนี้องค์ประกอบไม่ต้องมีอัตราส่วนแน่นอน และองค์ประกอบสามารถแยกออกยากกันได้ง่าย ตัวอย่าง

* น้ำกับน้ำมัน
* น้ำตาลกับเกลือ
* อากาศ
* เหล็กกับกำมะถัน
* ทรายและน้ำ

สารประกอบต่างจากสารผสมดังต่อไปนี้

1. คุณสมบัติของสารประกอบต่างจากขององค์ประกอบ ในขณะที่สารผสมมีคุณสมบัติขององค์ประกอบอยู่
1. อัตราส่วน สารประกอบมีอัตราส่วนที่แน่นอน
1. อะตอม สารประกอบอะตอมเชื่อกันด้วยพันธะเคมี
1. พลังงานความร้อนขณะเกิดสาร การเกิดสารประกอบเกิดพลังงานความร้อน

#### สารละลาย

> [สารละลาย](https://th.wikipedia.org/wiki/สารละลาย)คือสารผสมที่มีสารหนึ่งถูกละลาย

สารละลายประกอบด้วยตัวทำละลาย (Solvent) และตัวถูกละลาย (Solute) 

$$\rm{solvent} + \rm{solute} = \rm{solution}$$

#### สารแขวนลอย

> [สารแขวนลอย](https://th.wikipedia.org/wiki/สารแขวนลอย)คือส่วนผสมของของเหลวกับของแข็งที่ไม่ละลายในของเหลว

ของแข็งอาจสามารถแยกจากของเหลวโดยการกรอง

#### การทำละลาย

ถ้าวัสดุสามารถละลายได้ในวัสดุของเหลว วัสดุนั้นเรียกว่าทำละลายได้ ณร อุณหภูมิที่กำหนดของแข็งนั้นถึงจุดอิ่มตัวที่ไม่สามารถละลายต่อในของเหลวได้

> สารละลายถึงจุดอิ่มตัวเมื่อตัวถูกละลายไม่สามารถละลายต่อในตัวทำละลายต่อไปได้ โดยที่อุณหภูมิยังคงที่อยู่


> [การละลาย](https://th.wikipedia.org/wiki/การละลาย)เป็นการวัดปริมาณที่มากที่สุดของตัวถูกละลายที่สามารถทำละลายได้ในตัวทำละลาย 0.1 กิโลกรัม (100 กรัม) ณ ที่อุณหภูมิที่กำหนด

__ตัวอย่าง__ การละลายของโปแตสเซียมคลอไรด์ที่อุณภูุมิ 20 $^oC$ คือ 34 กรัมเปอร์เซ็นต์ของการทำละลายจึงคือ 34%

* การละลายขึ้นกับอุณหภูมิ
    * น้ำตาลละลาบในน้ำดีขึ้นเมื่ออุณหภูมิสูงขึ้น
    * เกลือแกงละลายได้ในน้ำเกือบคงที่ ที่อุณหภูมิสูงขึ้น
    * แคลเซียมไฮดรอกไซด์ละลายได้น้อยลงเมื่ออุณภูมิสูงขึ้น
* การละลายดีขึ้นเมื่อตัวถูกละลายมีขนาดเล็กลง
* การละลายดีขึ้นเมื่อมีการคนหรือทำการเขย่า

### ผลึก

[ผลึก (Crystals)](https://th.wikipedia.org/wiki/ผลึก) เป็นการจัดเรียงตัวอย่างมีรูปแบบของอะตอมหรือโมเลกุล จัดเรียงตัวกันจนเป็นหน่วยพื้นฐานของสสาร ของแข็งส่วนใหญ่โครงสร้างมีการจัดเรียงตัวเป็นผลึก ตัวอย่างเช่น กลือ น้ำตาล และโลหะ เป็นต้น

สารที่ไม่มีการจัดเรียงตัวเป็นผลึกเรียกว่าอสัณฐาน (Amorphous) ตัวอย่างเช่น แก้ว และไม้

การทำใหเกิดผลึกหรือเรียกว่าการตกผลึกเป็นขบวนการหนึ่งในการแยกของแข็งออกจากของเหลว

ผลึกสามารถมีขนาดต่างกัน แต่มี[โครงสร้างผลึก](https://th.wikipedia.org/wiki/โครงสร้างผลึก) รูปร่างทางเรขาคณิตเป็นหน้าแผ่นเรียบ ขอบตรง และมีมุมเฉพาะระหว่างด้าน ตัวอย่างรูปร่างโดยทั่วไปคือตามรูป

![ผลึก](U12pics/02.jpg)

ผลึกของเกลือแกง

![ผลึกเกลือแกง](U12pics/03.jpg)

### โลหะ

โลหะเป็นโพลีคริสตัลไลน์ (Polycrystalline) นั่นหมายความว่าโลหะประกอบด้วยผลึกจำนวนมาก เชื่อมต่อกันที่ขอบของผลึก ยิ่งมีขอบของผลึกมากเท่าไหร่วัสดุจะแข็งแรงมากขึ้น

ทุกโลหะเป็นของแข็งที่มีโครงสร้างผลึกของตัวเอง อัลลอยเป็นการผสมของโลหะต่างชนิดโดยการหลอมเหลว เมื่อหลอมเหลวแล้วทำให้ไม่มีความเป็นผลึก ของเหลวที่ได้ถูกทำให้เย็นลงและแข็งตัว ของแข็งที่ได้ประกอบไปด้วยผลึกต่างชนิดกัน อัลลอยจึงเป็นสารละลายที่เป็นของแข็ง ตัวอย่าง

* [ทองเหลือง](https://th.wikipedia.org/wiki/ทองเหลือง) มาจากทองแดงและสังกะสี
* [เหล็กกล้าไร้สนิม](https://th.wikipedia.org/wiki/เหล็กกล้าไร้สนิม) มาจากเหล็กและคาร์บอน
* [สัมฤทธิ์หรือสำริด](https://th.wikipedia.org/wiki/สัมฤทธิ์) มาจากทองแดงกับตะกั่ว

โลหะสามารถทำให้แข็งขึ้นโดยกานำไปเผาแล้วปล่อยให้เย็นตัวลงอย่างรวดเร็ว และสามารถทำให้อ่อนตัวลงโดยการนำไปเผาแล้วปล่อยให้เย็นตัวลงอย่างช้า ๆ หรือเรียกว่า[การอบเหนียว](https://en.wikipedia.org/wiki/Annealing_(metallurgy))

# ปริมาณสารสัมพันธ์

## มวลอะตอมและหน่วยมวลอะตอม และไอโซโทป

### เลขมวล

* เลขมวล (Mass Number $\rm{A}$) คือผลของจำนวนโปรตอนและนิวตรอนในนิวเคลียสของอะตอมหรือเรียกว่า นิวคลีออน (Nucleon)
* เลขอะตอม (Atomic Number $\rm{Z}$) คือจำนวนโปรตอนในนิวคลีออน
* สัญลักษณ์นิวเคลียร์แทนอะตอมของธาตุ $\rm{X}$ เป็น $_{\rm{Z}}^{\rm{A}}\rm{X}$ ตัวอย่าง
    * $_{\rm{6}}^{\rm{12}}\rm{C}$
    * $_{\rm{11}}^{\rm{23}}\rm{Na}$

### ไอโซโทป

* [ไอโซโทป](https://th.wikipedia.org/wiki/ไอโซโทป) คือธาตุชนิดเดียวกันคือมีจำนวนโปรตอนเท่ากัน แต่มีจำนวนนิวตรอนไมเท่ากัน
* ตัวอย่าง ไอโซโทปของธาตุไฮโดรเจน ![ไอโซโทป](https://upload.wikimedia.org/wikipedia/commons/6/6c/Protium_deuterium_tritium.jpg)
* [ตารางไอโซโทป](https://th.wikipedia.org/wiki/ตารางไอโซโทป_(แบ่งส่วน))

### มวลอะตอม

* เทียบกับ[หน่วยมวลอะตอม](https://th.wikipedia.org/wiki/หน่วยมวลอะตอม) นั่นคือเทียบกับ $\dfrac{1}{12}$ ของมวลของ $_{\rm{6}}^{\rm{12}}\rm{C}$ สัญลักษณ์ของหน่วยนี้ u หรือ Da (dalton)
* ไฮโดรเจน 1 อะตอมมีมวลประมาณ 1 u
* สัญลักษณ์ของหน่วยอะตอม amu ย่อมาจาก atomic mass unit ยังมีใช้ในงานตีพิมพ์เก่า ๆ โดยทั่วไปหน่วยมวลอะตอมนี้จะเขียนโดยไม่มีหน่วยกำกับ
*  $1 {\rm{\;u}} \approx 1.66053886\times 10^{-24} \rm{g}$
* บางครังเรียกมวลอะตอมเป็นน้ำหนักอะตอม 

### มวลอะตอมเฉลี่ย

* เนื่องจากอะตอมของธาตุในธรรมชาติมีหลายไอโซโทป จึงจำเป็นต้องหามวลอะตอมเฉลี่ย
* มวลอะตอมเฉลี่ย
$$\rm{average\;atomic\;mass} = \dfrac{\sum \rm{\%\,isotope\;abundant \times
atomic\;mass}}{100}$$
* [ตารางธาตุพร้อมชื่อและน้ำหนักอะตอมใหญ่](https://th.wikipedia.org/wiki/ตารางธาตุพร้อมชื่อและน้ำหนักอะตอมใหญ่)

__ตัวอย่าง__ ธาตุนีออน $_{10}\rm{Ne}$ ที่พบในธรรมชาติืมี 3 ไอโซโทป

* ชนิดที่ 1 มี 90.92% มีมวลเท่ากับ 19.99244
* ชนิดที่ 2 มี 0.257% มีมวลเท่ากับ 20.99395
* ชนิดที่ 3 มี 8.82%  $\;\,$มีมวลเท่ากับ 21.99138

In [8]:
import numpy as np

Neไอโซโทป = np.array([90.92, 0.257, 8.82])
Neมวล = np.array([19.99244, 20.99395, 21.99138])

มวลเฉลี่ย = np.sum(Neไอโซโทป*Neมวล)/100
print('น้ำหนักอะตอมเฉลี่ยของไอโซโทปทั้ง 3 ของนืออนคือ %2.3f'%มวลเฉลี่ย)

น้ำหนักอะตอมเฉลี่ยของไอโซโทปทั้ง 3 ของนืออนคือ 20.171


## เลขอาโวกาโดรและโมล

### เลขอาโวกาโดร

* $1 {\rm{\;u}} \approx 1.66053886\times 10^{-24} \rm{g}$

In [10]:
amu = 1.66053886*10**(-24)
print('%3.3E'%(1/amu))

6.022E+23


* ธาตุจำนวน $6.02\times 10^{23}$ อะตอมจะมีมวลเท่ากัยมวลอะตอมในหน่วยกรัม

### โมล

* โมลคือหน่วยบอกปริมาณสารที่มีจำนวนอนุภาค (อะตอม ไออน โมเลกุล) เท่ากับจำนวน $6.02\times 10^{23}$ อนุภาค
* ธาตุ 1 โมลอะตอมมีจำนวน $6.02\times 10^{23}$ อะตอม และหนักเท่ากับมวลอะตอมของธาตุนั้นในหน่วยกรัม
* สาร 1 โมลโมเลกุลมีจำนวน $6.02\times 10^{23}$ โมเลกุล และหนักเท่ากับมวลโมเลกุลของสารนั้นในหน่วยกรัม

ดังนั้น

$$n = \dfrac{N}{N_A} = \dfrac{m}{M}$$

เมื่อ

* $n$ คือจำนวนโมล
* $์N$ คือจำนวนอะตอมหรือโมเลกุล
* $N_A$ คือเลขอาโวกาโดรมีค่าเท่ากับ $6.02\times 10^{23}$
* $m$ คือน้ำหนักเป็นกรัม
* $M$ คือหน้ำหนักอะตอมหรือน้ำหนักโมเลกุล


* สำหรับแก๊สที่อุณหภูมิและความดันมาตรฐาน (STP ความดัน 1 บรรยากาศและอุณหภูมิ 0 $\rm{^oC}$) มีปริมาตร 22.4 ลิตร
$$ n = \dfrac{N}{N_0} = \dfrac{m}{M} = \dfrac{V}{V_0} \; \rm{(STP)}$$

## สมการเคมี

## สูตรและร้อยละขององค์ประกอบ

## ปริมาณสารสัมพันธ์ในสมการเคมี